tools/velvet/velveth.xml

```diff
@@ -29,6 +29,66 @@
       <option value="25">25</option>
       <option value="27">27</option>
       <option value="29">29</option>
+      <option value="31">31</option>
+      <option value="33">33</option>
+      <option value="35">35</option>
+      <option value="37">37</option>
+      <option value="39">39</option>
+      <option value="41">41</option>
+      <option value="43">43</option>
+      <option value="45">45</option>
+      <option value="47">47</option>
+      <option value="49">49</option>
+      <option value="51">51</option>
+      <option value="53">53</option>
+      <option value="55">55</option>
+      <option value="57">57</option>
+      <option value="59">59</option>
+      <option value="61">61</option>
+      <option value="63">63</option>
+      <option value="65">65</option>
+      <option value="67">67</option>
+      <option value="69">69</option>
+      <option value="71">71</option>
+      <option value="73">73</option>
+      <option value="75">75</option>
+      <option value="77">77</option>
+      <option value="79">79</option>
+      <option value="81">81</option>
+      <option value="83">83</option>
+      <option value="85">85</option>
+      <option value="87">87</option>
+      <option value="89">89</option>
+      <option value="91">91</option>
+      <option value="93">93</option>
+      <option value="95">95</option>
+      <option value="97">97</option>
+      <option value="99">99</option>
+      <option value="101">101</option>
+      <option value="103">103</option>
+      <option value="105">105</option>
+      <option value="107">107</option>
+      <option value="109">109</option>
+      <option value="111">111</option>
+      <option value="113">113</option>
+      <option value="115">115</option>
+      <option value="117">117</option>
+      <option value="119">119</option>
+      <option value="121">121</option>
+      <option value="123">123</option>
+      <option value="125">125</option>
+      <option value="127">127</option>
+      <option value="129">129</option>
+      <option value="131">131</option>
+      <option value="133">133</option>
+      <option value="135">135</option>
+      <option value="137">137</option>
+      <option value="139">139</option>
+      <option value="141">141</option>
+      <option value="143">143</option>
+      <option value="145">145</option>
+      <option value="147">147</option>
+      <option value="149">149</option>
```

Comment:
You can actually go until 191 https://github.com/bioconda/bioconda-recipes/blob/master/recipes/velvet/build.sh#L3 
if you depend on the bioconda package. This is a compile-time option.